playbooks/README.md

```diff
@@ -132,6 +132,13 @@ create the storage class.
 **Cinder**
 
 
+### Disable SELinux Playbook
+Disable SELinux on all hosts (Temporary playbook).
```

Comment:
It's not clear when it should be executed.

Reply:
And I think it can added simply as a Note:

```
>  **Note:** Text here. E.g.: Before installing KubeVirt on an existing OpenShift cluster, ensure that SELinux is disabled on all hosts: code here
```

Reply:
Agreed that this could be handled by docs, but folks want to have a playbook they can run when setting up a demo environment.  This playbook will temporarily live here.

Reply:
@rthallisey I didn't mean replace playbook by doc. I meant update doc to document the usage of that temporary playbook. The current doc string in this PR is not sufficient.  =)) Please reread ^^

Reply:
Line: 
```
  >  **Note:** Before installing KubeVirt on an existing OpenShift cluster, ensure that SELinux is disabled on all hosts:
$ ansible-playbook -i inventory playbooks/selinux.yml

```
should be placed before line: 

https://github.com/rthallisey/kubevirt-ansible/blame/4e8b81dd80e608a76668228bbb7b3de99664d4fe/playbooks/README.md#L103

If I understand it correctly

Reply:
sorry, if I wasn't clear.

Reply:
will update, thanks @alexxa